### ISSUE TITLE
Version 1.15 Fails to build because missing xfce4-hamster-plugin.pot file

### ISSUE THREAD

**Issue by darkshram** [projecthamster/xfce4-hamster-plugin#44]
- gettext 0.21
- cmake 3.20.6
- intltool 0.51.0
- glib2 2.66.8
- gtk3 3.24.31
- xfce4-panel-devel 4.16.3
- libxfce4ui-devel 4.16.1
- libxfce4util-devel 4.16.0
- xfconf-devel 4.16.0

```
[ 63%] Generating cs.gmo
/usr/bin/msgmerge --quiet --update --backup=none -s /builddir/build/BUILD/xfce4-hamster-plugin-1.15/po/cs.po /builddir/build/BUILD/xfce4-hamster-plugin-1.15/po/xfce4-hamster-plugin.pot
/usr/bin/msgmerge: error while opening "/builddir/build/BUILD/xfce4-hamster-plugin-1.15/po/xfce4-hamster-plugin.pot" for reading: No such file or directory
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu'
gmake[2]: *** [CMakeFiles/translations_1.dir/build.make:82: cs.gmo] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:106: CMakeFiles/translations_1.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
[ 68%] Building C object panel-plugin/CMakeFiles/hamster.dir/windowserver.c.o
cd /builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu/panel-plugin && /usr/lib64/ccache/cc -DGETTEXT_PACKAGE=\"xfce4-hamster-plugin\" -DLOCALEDIR=\"/usr/share/locale\" -Dhamster_EXPORTS -I/usr/include/xfce4/libxfce4panel-2.0 -I/usr/include/xfce4/libxfce4ui-2 -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/gio-unix-2.0 -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/fribidi -I/usr/include/atk-1.0 -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libdrm -I/usr/include/valgrind -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/xfce4 -I/usr/include/xfce4/xfconf-0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu/panel-plugin -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -fPIC -MD -MT panel-plugin/CMakeFiles/hamster.dir/windowserver.c.o -MF CMakeFiles/hamster.dir/windowserver.c.o.d -o CMakeFiles/hamster.dir/windowserver.c.o -c /builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu/panel-plugin/windowserver.c
xgettext: error while opening "./POTFILES.in.temp" for reading: No such file or directory
ERROR: xgettext failed to generate PO template file. Please consult
       error message above if there is any.
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu'
gmake[2]: *** [CMakeFiles/data.dir/build.make:83: ../po/xfce4-hamster-plugin.pot] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:158: CMakeFiles/data.dir/all] Error 2
[ 73%] Linking C shared library libhamster.so
cd /builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu/panel-plugin && /usr/bin/cmake -E cmake_link_script CMakeFiles/hamster.dir/link.txt --verbose=1
/usr/lib64/ccache/cc -fPIC -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -Wl,-z,relro -shared -Wl,-soname,libhamster.so -o libhamster.so CMakeFiles/hamster.dir/button.c.o CMakeFiles/hamster.dir/hamster.c.o CMakeFiles/hamster.dir/plugin.c.o CMakeFiles/hamster.dir/settings.c.o CMakeFiles/hamster.dir/util.c.o CMakeFiles/hamster.dir/view.c.o CMakeFiles/hamster.dir/windowserver.c.o  -lxfce4panel-2.0 -lgmodule-2.0 -lxfce4ui-2 -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lxfce4util -lxfconf-0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu'
[ 73%] Built target hamster
gmake[1]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.15/x86_64-redhat-linux-gnu'
gmake: *** [Makefile:139: all] Error 2
```

**Comment by aquaherd** [comment]
The pot file is created during compile and not checked in. Can I see your spec file? See APKBUILD for reference.

'''
# Maintainer: Hakan Erduman <hakan@erduman.de>
pkgname=xfce4-hamster-plugin
pkgver=1.15
pkgrel=0
pkgdesc="A recreation of the gnome-shell hamster extension as a xfce4 panel plugin."
url="https://github.com/projecthamster/xfce4-hamster-plugin"
arch="all !mips !mips64"
license="GPL-2.0-or-later"
options="!check" # No test suite
makedepends="cmake libxfce4ui-dev libxfce4util-dev xfce4-panel-dev"
subpackages="$pkgname-lang"
source="$pkgname-$pkgver.tar.gz::https://github.com/projecthamster/xfce4-hamster-plugin/archive/refs/tags/v$pkgver.tar.gz"

build() {
	cmake \
		-DCMAKE_INSTALL_PREFIX=/usr \
		-DCMAKE_INSTALL_LIBDIR=lib \
		-DCMAKE_BUILD_TYPE=None \
		.
	make
}

package() {
	make install LIBDIR=lib DESTDIR="$pkgdir"
}
'''

**Comment by darkshram** [comment]
```
Name:           xfce4-hamster-plugin
Version:        1.15
Release:        1%{?dist}
Summary:        Time tracker port of the 'hamster project extension' for the xfce4 panel

License:        GPLv2+
URL:            https://github.com/projecthamster/xfce4-hamster-plugin
Source0:        https://github.com/projecthamster/xfce4-hamster-plugin/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz

BuildRequires:  gcc
BuildRequires:  cmake
BuildRequires:  gettext
BuildRequires:  intltool
BuildRequires:  make
BuildRequires:  pkgconfig(gio-2.0)
BuildRequires:  pkgconfig(gio-unix-2.0)
BuildRequires:  pkgconfig(glib-2.0)
BuildRequires:  pkgconfig(gmodule-2.0)
BuildRequires:  pkgconfig(gobject-2.0)
BuildRequires:  pkgconfig(gthread-2.0)
BuildRequires:  pkgconfig(gtk+-3.0)
BuildRequires:  pkgconfig(libxfce4panel-2.0)
BuildRequires:  pkgconfig(libxfce4ui-2)
BuildRequires:  pkgconfig(libxfce4util-1.0)
BuildRequires:  pkgconfig(libxfconf-0)

Requires:       hamster-time-tracker >= 3.0.1

%description
This is a recreation of the gnome-shell hamster extension
as a xfce4 panel plugin.
See also the hamster project: <https://github.com/projecthamster>

%prep
%autosetup -p1 -n %{name}-%{version}

%build
%cmake
%cmake_build

%install
%cmake_install

%find_lang %{name}

%files -f %{name}.lang
%license COPYING
%doc AUTHORS ABOUT-NLS ChangeLog NEWS README*
%{_bindir}/xfce4-popup-hamstermenu
%{_libdir}/xfce4/panel/plugins/libhamster.so
%{_datadir}/xfce4/panel/plugins/hamster.desktop

%changelog
* Tue Dec 28 2021 Joel Barrios <http://www.alcancelibre.org/> - 1.15-1
- Update to 1.15.
```

**Comment by aquaherd** [comment]
Probably cmake threading issue. Potfile is generated but in parallel. 

- Also seen failing in AUR on First Build but succeeding in second. 
- Also seen failing in alpine CI in all but i386. 

Will fix asap.

**Comment by darkshram** [comment]
Did not work. I had to manually generate the pot file before %cmake to get past the build process.

```msgfilter -i po/es.po -o po/xfce4-hamster-plugin.pot true```

Without this adjustment, the build process fails with the following output:

```
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.XCt7pG
+ umask 022
+ cd /builddir/build/BUILD
+ cd /builddir/build/BUILD
+ rm -rf xfce4-hamster-plugin-1.16
+ /usr/bin/gzip -dc /builddir/build/SOURCES/xfce4-hamster-plugin-1.16.tar.gz
+ /bin/tar -xf -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd xfce4-hamster-plugin-1.16
+ /bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.5OFTsm
+ umask 022
+ cd /builddir/build/BUILD
+ cd xfce4-hamster-plugin-1.16
+ CFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic'
+ export CFLAGS
+ CXXFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic'
+ export CXXFLAGS
+ FFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -I/usr/lib64/gfortran/modules'
+ export FFLAGS
+ FCFLAGS='-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -I/usr/lib64/gfortran/modules'
+ export FCFLAGS
+ LDFLAGS='-Wl,-z,relro '
+ export LDFLAGS
+ /usr/bin/cmake -S . -B x86_64-redhat-linux-gnu -DCMAKE_C_FLAGS_RELEASE:STRING=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE:STRING=-DNDEBUG -DCMAKE_Fortran_FLAGS_RELEASE:STRING=-DNDEBUG -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF -DCMAKE_INSTALL_PREFIX:PATH=/usr -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DLIB_SUFFIX=64 -DBUILD_SHARED_LIBS:BOOL=ON
-- The C compiler identification is GNU 7.3.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib64/ccache/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1") 
-- Found Gettext: /usr/bin/msgmerge (found version "0.21") 
-- Found Intl: built in to C library  
-- Checking for modules 'libxfce4panel-2.0;libxfce4ui-2;libxfconf-0'
--   Found libxfce4panel-2.0, version 4.16.3
--   Found libxfce4ui-2, version 4.16.1
--   Found libxfconf-0, version 4.16.0
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_CXX_FLAGS_RELEASE
    CMAKE_C_FLAGS_RELEASE
    CMAKE_Fortran_FLAGS_RELEASE
    CMAKE_INSTALL_DO_STRIP
    INCLUDE_INSTALL_DIR
    LIB_INSTALL_DIR
    LIB_SUFFIX
    SHARE_INSTALL_PREFIX
    SYSCONF_INSTALL_DIR


-- Build files have been written to: /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu
+ /usr/bin/cmake --build x86_64-redhat-linux-gnu -j8 --verbose
/usr/bin/cmake -S/builddir/build/BUILD/xfce4-hamster-plugin-1.16 -B/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu/CMakeFiles /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu//CMakeFiles/progress.marks
/usr/bin/gmake  -f CMakeFiles/Makefile2 all
gmake[1]: Entering directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f CMakeFiles/translations_1.dir/build.make CMakeFiles/translations_1.dir/depend
/usr/bin/gmake  -f CMakeFiles/data.dir/build.make CMakeFiles/data.dir/depend
gmake[2]: Entering directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
cd /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /builddir/build/BUILD/xfce4-hamster-plugin-1.16 /builddir/build/BUILD/xfce4-hamster-plugin-1.16 /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu/CMakeFiles/translations_1.dir/DependInfo.cmake --color=
gmake[2]: Entering directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
cd /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /builddir/build/BUILD/xfce4-hamster-plugin-1.16 /builddir/build/BUILD/xfce4-hamster-plugin-1.16 /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu/CMakeFiles/data.dir/DependInfo.cmake --color=
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
/usr/bin/gmake  -f CMakeFiles/translations_1.dir/build.make CMakeFiles/translations_1.dir/build
/usr/bin/gmake  -f CMakeFiles/data.dir/build.make CMakeFiles/data.dir/build
gmake[2]: Entering directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
gmake[2]: Entering directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
[  5%] Generating ../po/xfce4-hamster-plugin.pot
[ 15%] Generating hamster.desktop
[ 15%] Generating ../po/xfce4-hamster-plugin.pot
cd /builddir/build/BUILD/xfce4-hamster-plugin-1.16/po && /usr/bin/intltool-update -p -g xfce4-hamster-plugin
/usr/bin/intltool-merge -d /builddir/build/BUILD/xfce4-hamster-plugin-1.16/po /builddir/build/BUILD/xfce4-hamster-plugin-1.16/panel-plugin/hamster.desktop.in /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu/hamster.desktop
cd /builddir/build/BUILD/xfce4-hamster-plugin-1.16/po && /usr/bin/intltool-update -p -g xfce4-hamster-plugin
Merging translations into /builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu/hamster.desktop.
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
xgettext: error while opening "./POTFILES.in.temp" for reading: No such file or directory
ERROR: xgettext failed but still managed to generate PO template file.
       Please consult error message above if there is any.
gmake[2]: *** [CMakeFiles/translations_1.dir/build.make:122: ../po/xfce4-hamster-plugin.pot] Error 1
gmake[2]: *** Deleting file '../po/xfce4-hamster-plugin.pot'
gmake[2]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
gmake[1]: *** [CMakeFiles/Makefile2:106: CMakeFiles/translations_1.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
[ 15%] Built target data
gmake[1]: Leaving directory '/builddir/build/BUILD/xfce4-hamster-plugin-1.16/x86_64-redhat-linux-gnu'
gmake: *** [Makefile:139: all] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.5OFTsm (%build)
    Bad exit status from /var/tmp/rpm-tmp.5OFTsm (%build)
```
Executing msgfilter before cmake to generate the pot file let me pass the build process, but then fails in the install process because libhamster.so installs as /usr/lib/xfce4/panel/plugins/libhamster.so instead of /usr/lib64/xfce4/panel/plugins/libhamster.so becuase it ignores the values LIB_INSTALL_DIR and LIB_SUFFIX from cmake macros. So version 1.15/1.16 is not multilib-aware.

```
Executing(%install): /bin/sh -e /var/tmp/rpm-tmp.VCgAIg
+ umask 022
+ cd /builddir/build/BUILD
+ '[' /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64 '!=' / ']'
+ rm -rf /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64
++ dirname /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64
+ mkdir -p /builddir/build/BUILDROOT
+ mkdir /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64
+ cd xfce4-hamster-plugin-1.16
+ DESTDIR=/builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64
+ /usr/bin/cmake --install x86_64-redhat-linux-gnu
-- Install configuration: ""
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/locale/cs/LC_MESSAGES/xfce4-hamster-plugin.mo
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/locale/de/LC_MESSAGES/xfce4-hamster-plugin.mo
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/locale/es/LC_MESSAGES/xfce4-hamster-plugin.mo
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/locale/ru/LC_MESSAGES/xfce4-hamster-plugin.mo
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/locale/tr/LC_MESSAGES/xfce4-hamster-plugin.mo
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/locale/uk/LC_MESSAGES/xfce4-hamster-plugin.mo
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/lib/xfce4/panel/plugins/libhamster.so
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/xfce4/panel/plugins/hamster.desktop
-- Installing: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/bin/xfce4-popup-hamstermenu
+ /usr/lib/rpm/find-lang.sh /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64 xfce4-hamster-plugin
+ /usr/lib/rpm/find-debuginfo.sh --strict-build-id -m --run-dwz --dwz-low-mem-die-limit 10000000 --dwz-max-die-limit 110000000 /builddir/build/BUILD/xfce4-hamster-plugin-1.16
extracting debug info from /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/lib/xfce4/panel/plugins/libhamster.so
dwz: Too few files for multifile optimization
/usr/lib/rpm/sepdebugcrcfix: Updated 1 CRC32s, 0 CRC32s did match.
652 blocks
+ /usr/lib/rpm/check-buildroot
+ /usr/lib/rpm/redhat/brp-compress
+ /usr/lib/rpm/redhat/brp-strip-static-archive /usr/bin/strip
+ /usr/lib/rpm/brp-python-bytecompile /usr/bin/python 1
+ /usr/lib/rpm/redhat/brp-python-hardlink
+ /usr/lib/rpm/redhat/brp-java-repack-jars
Processing files: xfce4-hamster-plugin-1.16-1.fc14.al.x86_64
error: File not found: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/lib64/xfce4/panel/plugins/libhamster.so
Executing(%doc): /bin/sh -e /var/tmp/rpm-tmp.4aqzNo
+ umask 022
+ cd /builddir/build/BUILD
+ cd xfce4-hamster-plugin-1.16
+ DOCDIR=/builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/doc/xfce4-hamster-plugin-1.16
+ export DOCDIR
+ /bin/mkdir -p /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/doc/xfce4-hamster-plugin-1.16
+ cp -pr AUTHORS /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/doc/xfce4-hamster-plugin-1.16
+ cp -pr NEWS /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/doc/xfce4-hamster-plugin-1.16
+ cp -pr README README.md /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/doc/xfce4-hamster-plugin-1.16
+ exit 0
Executing(%license): /bin/sh -e /var/tmp/rpm-tmp.DUBHTw
+ umask 022
+ cd /builddir/build/BUILD
+ cd xfce4-hamster-plugin-1.16
+ LICENSEDIR=/builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/licenses/xfce4-hamster-plugin-1.16
+ export LICENSEDIR
+ /bin/mkdir -p /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/licenses/xfce4-hamster-plugin-1.16
+ cp -pr COPYING /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/share/licenses/xfce4-hamster-plugin-1.16
+ exit 0


RPM build errors:
    File not found: /builddir/build/BUILDROOT/xfce4-hamster-plugin-1.16-1.fc14.al.x86_64/usr/lib64/xfce4/panel/plugins/libhamster.so
```
To build the rpm package, I had to use this spec file:
```
Name:           xfce4-hamster-plugin
Version:        1.16
Release:        1%{?dist}
Summary:        Time tracker port of the 'hamster project extension' for the xfce4 panel

License:        GPLv2+
URL:            https://github.com/projecthamster/xfce4-hamster-plugin
Source0:        https://github.com/projecthamster/xfce4-hamster-plugin/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz

BuildRequires:  gcc
BuildRequires:  cmake
BuildRequires:  gettext
BuildRequires:  intltool
BuildRequires:  make
BuildRequires:  pkgconfig(gio-2.0)
BuildRequires:  pkgconfig(gio-unix-2.0)
BuildRequires:  pkgconfig(glib-2.0)
BuildRequires:  pkgconfig(gmodule-2.0)
BuildRequires:  pkgconfig(gobject-2.0)
BuildRequires:  pkgconfig(gthread-2.0)
BuildRequires:  pkgconfig(gtk+-3.0)
BuildRequires:  pkgconfig(libxfce4panel-2.0)
BuildRequires:  pkgconfig(libxfce4ui-2)
BuildRequires:  pkgconfig(libxfce4util-1.0)
BuildRequires:  pkgconfig(libxfconf-0)

Requires:       hamster-time-tracker >= 3.0.1

%description
This is a recreation of the gnome-shell hamster extension as a xfce4
panel plugin. See also the hamster project:
<https://github.com/projecthamster>

%prep
%autosetup -p1 -n %{name}-%{version}
msgfilter -i po/es.po -o po/xfce4-hamster-plugin.pot true

%build
%cmake
%cmake_build

%install
%cmake_install

%ifarch x86_64
mkdir -p %{buildroot}%{_libdir}/xfce4/panel/plugins
mv %{buildroot}%{_usr}/lib/xfce4/panel/plugins/libhamster.so \
    %{buildroot}%{_libdir}/xfce4/panel/plugins/libhamster.so
%endif

%find_lang %{name}

%files -f %{name}.lang
%license COPYING
%doc AUTHORS NEWS README*
%{_bindir}/xfce4-popup-hamstermenu
%{_libdir}/xfce4/panel/plugins/libhamster.so
%{_datadir}/xfce4/panel/plugins/hamster.desktop

%changelog
* Wed Dec 29 2021 Joel Barrios <http://www.alcancelibre.org/> - 1.16-1
- Update to 1.16.
```
After installing, xfce4-hamster-plugin works perfectly as expected.

**Comment by aquaherd** [comment]
Please try again with v1.17 - should compile in multi core and honor libdir via prefix setting during install

**Comment by darkshram** [comment]
> Please try again with v1.17 - should compile in multi core and honor libdir via prefix setting during install

Already done. It works! Thank you! Builds without issue.

```
Name:           xfce4-hamster-plugin
Version:        1.17
Release:        1%{?dist}
Summary:        Time tracker port of the 'hamster project extension' for the xfce4 panel

License:        GPLv2+
URL:            https://github.com/projecthamster/xfce4-hamster-plugin
Source0:        https://github.com/projecthamster/xfce4-hamster-plugin/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz

BuildRequires:  gcc
BuildRequires:  cmake
BuildRequires:  gettext
BuildRequires:  gettext-devel
BuildRequires:  intltool
BuildRequires:  make
BuildRequires:  pkgconfig(gio-2.0)
BuildRequires:  pkgconfig(gio-unix-2.0)
BuildRequires:  pkgconfig(glib-2.0)
BuildRequires:  pkgconfig(gmodule-2.0)
BuildRequires:  pkgconfig(gobject-2.0)
BuildRequires:  pkgconfig(gthread-2.0)
BuildRequires:  pkgconfig(gtk+-3.0)
BuildRequires:  pkgconfig(libxfce4panel-2.0)
BuildRequires:  pkgconfig(libxfce4ui-2)
BuildRequires:  pkgconfig(libxfce4util-1.0)
BuildRequires:  pkgconfig(libxfconf-0)

Requires:       hamster-time-tracker >= 3.0.1

%description
This is a recreation of the gnome-shell hamster extension as a xfce4
panel plugin. See also the hamster project:
<https://github.com/projecthamster>

%prep
%autosetup -p1

%build
%cmake
%cmake_build

%install
%cmake_install
%find_lang %{name}

%files -f %{name}.lang
%license COPYING
%doc AUTHORS NEWS README.md
%{_bindir}/xfce4-popup-hamstermenu
%{_libdir}/xfce4/panel/plugins/libhamster.so
%{_datadir}/xfce4/panel/plugins/hamster.desktop

%changelog
* Sat Jan 01 2022 Joel Barrios <http://www.alcancelibre.org/> - 1.17-1
- Update to 1.17.
```
